superset/config.py

```diff
@@ -258,6 +258,7 @@
 # method.
 # note: index option should not be overridden
 CSV_EXPORT = {"encoding": "utf-8"}
+CSV_EXPORT_VERBOSE = True
```

Comment:
This could be expressed more verbosely (pun unintended), like `CSV_EXPORT_USE_VERBOSE_NAMES_AS_HEADERS` or similar.